.github/workflows/acceptance-tests-runner.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Acceptance Tests'
+name: 'Acceptance Tests Execution'
```

Comment:
Could we add a [description](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#description) to our actions? I see how new joiners might be confused by their names

Reply:
good point, unfortunately workflow does not support top-level descriptions :/ (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions)
Moved a a comment clarifying the purpose to the top.

Reply:
Thanks! Do you mind adding a comment to other active actions as well or creating a ticket to do so?

Reply:
Included 👍